src/page-objects/administration/PaymentDetail.ts

```diff
@@ -0,0 +1,20 @@
+import type { Page, Locator } from '@playwright/test';
+import type { PageObject } from '../../types/PageObject';
+
+export class PaymentDetail implements PageObject {
+
+    public readonly header: Locator;
+    public readonly nameField: Locator;
+    public readonly availabilityRule: Locator;
+
+
+    constructor(public readonly page: Page) {
+        this.header = page.locator('.smart-bar__header');
+        this.nameField = page.getByLabel('Name').first();
```

Comment:
Why you took first() here? Is there a second one? if yes, its possible to make it unique somehow?

Reply:
The second one is the technical name field. But you were right, there is a better locator to pinpoint the actual name field. 👌